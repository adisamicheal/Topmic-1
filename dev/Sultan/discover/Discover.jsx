import React, { Component } from 'react';

export default class Feature extends Component {

  componentDidMount() {
    
  }

  render() {
    return (
      <div id="discover">
        <div className="Text">
          <h3 className="discover-heading">Discover More About Winsome</h3>
        </div>
        <div className="one-third events">
          <h3 className="discover-subheading">Events</h3>
        </div>
        <div className="one-third faculty">
          <h3 className="discover-subheading">Faculty & Staff</h3>
        </div>
        <div className="one-third policies">
          <h3 className="discover-subheading">Policies & Procedures</h3>
        </div>
        <div className="clear" />
      </div>
    )
  }
};