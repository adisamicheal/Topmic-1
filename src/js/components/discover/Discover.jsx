import React, { Component } from 'react';

export default class Feature extends Component {
  componentDidMount() {}

  render() {
    return (
      <div className="discover-container">
        <div className="header">
          <div className="header-content">
            <h3>Discover More About Winsome</h3>
          </div>
        </div>
        <div className="content">
          <div className="first">
            <div className="footer-discover">
              <h3>Events</h3>
            </div>
          </div>
          <div className="second">
            <div className="footer-discover">
              <h3>Faculty & Staff</h3>
            </div>
          </div>
          <div className="third">
            <div className="footer-discover">
              <h3>Policies & Procedures</h3>
            </div>
          </div>
        </div>
      </div>
    );
  }
}
