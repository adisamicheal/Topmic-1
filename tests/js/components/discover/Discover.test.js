import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import Discover from '../../../../src/js/components/discover/Discover';

Enzyme.configure({ adapter: new Adapter() });

describe('testing Discover component', () => {
  const wrapper = shallow(<Discover />);
  it('renders the Discover component', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
