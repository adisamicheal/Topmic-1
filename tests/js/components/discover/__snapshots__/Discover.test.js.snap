// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing Discover component renders the Discover component 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Feature />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="header"
        >
          <div
            className="header-content"
          >
            <h3>
              Discover More About Winsome
            </h3>
          </div>
        </div>,
        <div
          className="content"
        >
          <div
            className="first"
          >
            <div
              className="footer-discover"
            >
              <h3>
                Events
              </h3>
            </div>
          </div>
          <div
            className="second"
          >
            <div
              className="footer-discover"
            >
              <h3>
                Faculty & Staff
              </h3>
            </div>
          </div>
          <div
            className="third"
          >
            <div
              className="footer-discover"
            >
              <h3>
                Policies & Procedures
              </h3>
            </div>
          </div>
        </div>,
      ],
      "className": "discover-container",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <div
            className="header-content"
          >
            <h3>
              Discover More About Winsome
            </h3>
          </div>,
          "className": "header",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <h3>
              Discover More About Winsome
            </h3>,
            "className": "header-content",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Discover More About Winsome",
            },
            "ref": null,
            "rendered": "Discover More About Winsome",
            "type": "h3",
          },
          "type": "div",
        },
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div
              className="first"
            >
              <div
                className="footer-discover"
              >
                <h3>
                  Events
                </h3>
              </div>
            </div>,
            <div
              className="second"
            >
              <div
                className="footer-discover"
              >
                <h3>
                  Faculty & Staff
                </h3>
              </div>
            </div>,
            <div
              className="third"
            >
              <div
                className="footer-discover"
              >
                <h3>
                  Policies & Procedures
                </h3>
              </div>
            </div>,
          ],
          "className": "content",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <div
                className="footer-discover"
              >
                <h3>
                  Events
                </h3>
              </div>,
              "className": "first",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <h3>
                  Events
                </h3>,
                "className": "footer-discover",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Events",
                },
                "ref": null,
                "rendered": "Events",
                "type": "h3",
              },
              "type": "div",
            },
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <div
                className="footer-discover"
              >
                <h3>
                  Faculty & Staff
                </h3>
              </div>,
              "className": "second",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <h3>
                  Faculty & Staff
                </h3>,
                "className": "footer-discover",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Faculty & Staff",
                },
                "ref": null,
                "rendered": "Faculty & Staff",
                "type": "h3",
              },
              "type": "div",
            },
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <div
                className="footer-discover"
              >
                <h3>
                  Policies & Procedures
                </h3>
              </div>,
              "className": "third",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <h3>
                  Policies & Procedures
                </h3>,
                "className": "footer-discover",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Policies & Procedures",
                },
                "ref": null,
                "rendered": "Policies & Procedures",
                "type": "h3",
              },
              "type": "div",
            },
            "type": "div",
          },
        ],
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="header"
          >
            <div
              className="header-content"
            >
              <h3>
                Discover More About Winsome
              </h3>
            </div>
          </div>,
          <div
            className="content"
          >
            <div
              className="first"
            >
              <div
                className="footer-discover"
              >
                <h3>
                  Events
                </h3>
              </div>
            </div>
            <div
              className="second"
            >
              <div
                className="footer-discover"
              >
                <h3>
                  Faculty & Staff
                </h3>
              </div>
            </div>
            <div
              className="third"
            >
              <div
                className="footer-discover"
              >
                <h3>
                  Policies & Procedures
                </h3>
              </div>
            </div>
          </div>,
        ],
        "className": "discover-container",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <div
              className="header-content"
            >
              <h3>
                Discover More About Winsome
              </h3>
            </div>,
            "className": "header",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <h3>
                Discover More About Winsome
              </h3>,
              "className": "header-content",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Discover More About Winsome",
              },
              "ref": null,
              "rendered": "Discover More About Winsome",
              "type": "h3",
            },
            "type": "div",
          },
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <div
                className="first"
              >
                <div
                  className="footer-discover"
                >
                  <h3>
                    Events
                  </h3>
                </div>
              </div>,
              <div
                className="second"
              >
                <div
                  className="footer-discover"
                >
                  <h3>
                    Faculty & Staff
                  </h3>
                </div>
              </div>,
              <div
                className="third"
              >
                <div
                  className="footer-discover"
                >
                  <h3>
                    Policies & Procedures
                  </h3>
                </div>
              </div>,
            ],
            "className": "content",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <div
                  className="footer-discover"
                >
                  <h3>
                    Events
                  </h3>
                </div>,
                "className": "first",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <h3>
                    Events
                  </h3>,
                  "className": "footer-discover",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Events",
                  },
                  "ref": null,
                  "rendered": "Events",
                  "type": "h3",
                },
                "type": "div",
              },
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <div
                  className="footer-discover"
                >
                  <h3>
                    Faculty & Staff
                  </h3>
                </div>,
                "className": "second",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <h3>
                    Faculty & Staff
                  </h3>,
                  "className": "footer-discover",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Faculty & Staff",
                  },
                  "ref": null,
                  "rendered": "Faculty & Staff",
                  "type": "h3",
                },
                "type": "div",
              },
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <div
                  className="footer-discover"
                >
                  <h3>
                    Policies & Procedures
                  </h3>
                </div>,
                "className": "third",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <h3>
                    Policies & Procedures
                  </h3>,
                  "className": "footer-discover",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Policies & Procedures",
                  },
                  "ref": null,
                  "rendered": "Policies & Procedures",
                  "type": "h3",
                },
                "type": "div",
              },
              "type": "div",
            },
          ],
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
